### ISSUE TITLE
[cmake] python coverage (Trac #1135)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1129]
Add python coverage target.  It should run over all the python tests.

Migrated from https://code.icecube.wisc.edu/ticket/1135
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T23:59:06", 
    "description": "Add python coverage target.  It should run over all the python tests.", 
    "reporter": "david.schultz", 
    "cc": "nega", 
    "resolution": "fixed", 
    "_ts": "1547251146643413", 
    "component": "cmake", 
    "summary": "[cmake] python coverage", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-08-17T20:26:37", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150818T20:50:23: 
* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 2226:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2226"
> add python coverage targets (pycoverage and pycoverage_verbose). closes #1129
> ```

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:13:59: 
* olivas changed milestone from "offline V15-08-00" to ""
* olivas commented:

> Milestone offline V15-08-00 deleted

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:59:06: nwhitehorn commented:

> In changeset 170094:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170094"
> add python coverage targets (pycoverage and pycoverage_verbose). closes #1129
> ```
